Fabric.Identity.API/scripts/Install-Identity-Utilities.psm1

```diff
@@ -632,7 +632,61 @@ function Get-DefaultApplicationEndpoint([string] $appName, [string] $appEndPoint
 }
 
 function Get-FullyQualifiedMachineName() {
-	return "https://$env:computername.$((Get-WmiObject Win32_ComputerSystem).Domain.tolower())"
+    return "https://$env:computername.$((Get-WmiObject Win32_ComputerSystem).Domain.tolower())"
+}
+
+function Get-ApplicationUrl($serviceName, $discoveryServiceUrl){
```

Comment:
Does Dos-InstallUtilities have functions for this?

Reply:
It does not